packages/tools/devtools/devtools-core/src/ContainerDevtools.ts

```diff
@@ -90,10 +94,10 @@ export interface ContainerDevtoolsProps extends HasContainerKey {
  * - {@link GetContainerState.Message}: When received, {@link ContainerStateChange.Message} will be posted in response.
  *
  * - {@link ConnectContainer.Message}: When received, {@link @fluidframework/container-definitions#IContainer.connect}
- * will be called on the {@link ContainerDevtools.container} (if it is disconnected).
+ * will be called on the {@link ContainerDevtools.container} (if it is connected).
```

Comment:
The phrasing here was a bit ambiguous, but these updates aren't correct. The idea here is that clicking the "connect" button will only attempt to connect the container if it was _disconnected_ (calling connect on an already connected container won't do anything).

I would maybe update this to say `if it is currently disconnected`.